mass_mailing_editor_af/views/snippets_themes.xml

```diff
@@ -1185,6 +1185,50 @@
             </div>
         </div>
     </template>
+  
+    <!--textmodul med H3 och länkktext 2 kolumner-->
+    <template id="s_mail_block_toppyta_h3_text" name="H3 och länktext 2 kolumner">
+        <div class="snippets_div" style="max-width: 600px;background-color: white; padding: 8px 24px 24px 24px;">
+            <div style="display:inline-block ; width:260px; background-color: white; vertical-align: top;">
+                <h3 class="snippets_h1_dark" style="width: 260px;color: rgb(51, 51, 51);
+                    font-size: 20px; font-family: 'Open Sans Semibold'; font-weight: 600; letter-spacing: 0px; line-height: 30px;">
+                    H3 - vänster alignerad
+                </h3>
+                <p style="width: 260px; color: rgb(51, 51, 51); font-size: 16px; font-family: 'Open Sans'; letter-spacing: 0px; line-height: 24px;">
+                    Brödtext. Rekommendation 150-250 tecken.
+                </p>
+                <div style="color:#ffffff!important;">
+                    <div style="margin-left:4px"><img src="/mass_mailing_editor_af/static/src/img/blocks/arrow-darkblue.png"/>
```

Comment:
Use svg-format instead of png for the image. Change to IkonerChevron_Right_light.svg

I would also suggest to put width="8" inside the img-tag to get it consistent with other similar snippets
(img src="/mass_mailing_editor_af/static/src/img/blocks/IkonerChevron_Right_light.svg" width="8")